projectPython/remember_remember.py

```diff
@@ -0,0 +1,21 @@
+from PIL import Image
+
+
+def decoding_message_in_image(image_path):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/